pykechain/models/service.py

```diff
@@ -75,8 +75,14 @@ def execute(self, interactive: Optional[bool] = False) -> 'ServiceExecution':
         :return: ServiceExecution when successful.
         :raises APIError: when unable to execute
         """
+        request_params = dict(
+            interactive=check_type(interactive, bool, 'interactive'),
+            format='json',
+            **kwargs
+        )
+
         url = self._client._build_url('service_execute', service_id=self.id)
-        response = self._client._request('GET', url, params=dict(interactive=interactive, format='json'))
+        response = self._client._request('GET', url, params=request_params)
```

Comment:
Write a test that sends the `activity_id` (or whatever is needed) and check if the `ServiceExecution` has a context.

Reply:
Will do!